bokehjs/src/lib/models/layouts/tabs.ts

```diff
@@ -102,6 +112,24 @@ export class TabsView extends LayoutDOMView {
         if (event.target == event.currentTarget)
           this.change_active(i)
       })
+      const tooltip = this.tooltips[i]
+      if (tooltip) {
+        tooltip.useRelativeBoundingBox = false
+        tooltip?.model.setv({
+          target: el,
+        })
+        const toggleTooltip = (visible: boolean) => {
+          tooltip.model.setv({
+            visible,
+          })
+        }
+        el.addEventListener("mouseenter", () => {
+          toggleTooltip(true)
```

Comment:
We use `toggle_tooltip` naming convention for variables and functions. This needs to be changed across the PR.

Reply:
Thank you for pointing that out. This should be resolved across the PR now.